R/aaa-auto.R

```diff
@@ -391,6 +391,25 @@ simple_interconnected_islands_game_impl <- function(islands.n, islands.size, isl
   res
 }
 
+chung_lu_game_impl <- function(out.weights, in.weights=NULL, loops=TRUE, variant=c("original", "maxent", "nr")) {
```

Comment:
Do we want this?

```suggestion
chung_lu_game_impl <- function(out.weights, ..., in.weights=NULL, loops=TRUE, variant=c("original", "maxent", "nr")) {
```

Reply:
No, definitely not. The ellipsis should come _after_ `in.weights`, not before.